doc/source/api.rst

```diff
@@ -2328,3 +2565,77 @@ Scalar introspection
     api.types.is_re
     api.types.is_re_compilable
     api.types.is_scalar
+
+
+.. This is to prevent warnings in the doc build. We don't want to encourage
+.. these methods.
+
+.. toctree::
+   :hidden:
+
+   generated/pandas.DataFrame.as_blocks
+   generated/pandas.DataFrame.bfill
+   generated/pandas.DataFrame.blocks
+   generated/pandas.DataFrame.columns
+   generated/pandas.DataFrame.consolidate
+   generated/pandas.DataFrame.ffill
+   generated/pandas.DataFrame.get_value
+   generated/pandas.DataFrame.index
+   generated/pandas.DataFrame.ix
+   generated/pandas.DataFrame.set_value
+   generated/pandas.DataFrame.sortlevel
+
+   generated/pandas.Index.data.rst
+   generated/pandas.Index.asi8.rst
+   generated/pandas.Index.flags
+   generated/pandas.Index.isnull
+   generated/pandas.Index.names
+   generated/pandas.Index.nlevels
+   generated/pandas.Index.notnull
+   generated/pandas.Index.reshape
+   generated/pandas.Index.sort
+   generated/pandas.Index.sortlevel
+
+   generated/pandas.Index.str
+
+   generated/pandas.Panel.as_blocks.rst
+   generated/pandas.Panel.blocks.rst
+   generated/pandas.Panel.consolidate.rst
```

Comment:
Here (and below) also a bunch of .rst ones (it is a bit strange that this does not lead to warnings in the doc build .. ?)